### DISCUSSION TITLE
Add GUI for reviewing and approving changes

### DISCUSSION THREAD

**Discussion by SaraCooperQA** [simonsmith/cypress-image-snapshot#21] [General]
Hi @simonsmith . Great job on this plugin!  I do have a feature request if possible.

Add a graphical interface for reviewing and approving changes. It would show the diff image you create and at the top could be a button called Update Screenshot and a Close button. The Update button would make/update a base screenshot that would be used in future running of the test.

There is another plugin that has a similar feature, but that plugin has some issues for me. You can check it out to see the feature: https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff

Thanks

**Comment by simonsmith** [comment]
Interesting. Is this something that can only be used in open mode?
I'm curious because I typically generate my snapshots in run mode in Docker and not sure how this would work

**Reply by SaraCooperQA** [reply]
Yes, this would be for open mode. I run our tests in open mode daily. It would be a big time saver.

**Reply by simonsmith** [reply]
I see, so you would typically commit the snapshots generated in open mode? Not a workflow I'm familiar with but I can take a look at how it's implemented in that example you provided

**Reply by SaraCooperQA** [reply]
Thanks for taking a look. Let me know if you have any questions.

**Reply by donwelsh** [reply]
@simonsmith Have you had a chance to look into this, is this a feature you could implement. I could use this feature as well. Thanks!

**Reply by simonsmith** [reply]
I did look at this, and it didn't seem like a trivial feature to implement. I'm not against adding it but I personally never use open mode to actually generate snapshots (I do it in run mode via Docker) so it's not something I'd use.

Open to contributions though!
